reporting/data_xfer_by_month_no_bots.query.json

```diff
@@ -0,0 +1,61 @@
+{
+  "aggs": {
+    "Month": {
+      "date_histogram": {
+        "field": "@apache_timestamp",
+        "interval": "1M",
+        "time_zone": "America/New_York",
+        "min_doc_count": 1,
+        "format": "yyyy-MM-dd"
+      },
+      "aggs": {
+        "Bytes_transferred": {
+          "sum": {
+            "field": "bytes",
+            "script": {
+              "source" : "((_value / 1024) / 1024) / 1024"
```

Comment:
I don't know if I'm getting too deep into the weeds by commenting on this, but why is it `/ 1024` three times?

Reply:
The data is saved as number of bytes. Dividing by 1024 gives KB, dividing again gives MB, 
dividing again: GB. We want the resulting format to be gigabytes.

I thought about simplifying it, but when changing precision of number of bytes, I find it easier to understand when written this way.